docs/examples/polkadot/interface.sol

```diff
@@ -1,42 +1,39 @@
-interface operator {
-    function op1(int32 a, int32 b) external returns (int32);
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
```

Comment:
While for `solc` this is prefered. I think that the version pragma here should be removed from the example. It is purely a thing for Ethereum Soliditys `solc` compiler.  It specifies a Ethereum Solidity version, not a Solang version. As per [our docs](https://solang.readthedocs.io/en/v0.3.2/language/pragmas.html#pragmas):

> Solang does not follow the Ethereum Foundation Solidity compiler version numbering scheme, so these pragma statements are silently ignored.

Because this example is an example for Solidity on Polkadot (and not EVM), it is unnecessary: Having it in the example implies that this is required or good practice, which isn't the case.